src/dbus_api.rs

```diff
@@ -223,12 +223,12 @@ fn create_pool(m: &MethodInfo<MTFn<TData>, TData>) -> MethodResult {
 
     match result {
         Ok(_) => {
-            let dbus_contex_clone = dbus_context.clone();
-            let object_path_message_item = create_dbus_pool(dbus_contex_clone);
+            let dbus_context_clone = dbus_context.clone();
```

Comment:
I didn't touch this, because I didn't introduce that clone, and I don't know the reason for the clone in the first place.